packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -75,17 +75,17 @@ class WalletsController {
     name,
     password,
     mnemonic,
-    receiveAddressNumber = 17,
+    receivingAddressNumber: receivingAddressNumber = 17,
```

Comment:
Type should be named under the rule of PascalCase.

Reply:
Oops, not sure why type was added in the first place. Automatically added by format?